src/wallet/rpcwallet.cpp

```diff
@@ -1640,12 +1657,15 @@ UniValue listsinceblock(const JSONRPCRequest& request)
 
     if (request.fHelp)
         throw runtime_error(
-            "listsinceblock ( \"blockhash\" target_confirmations include_watchonly)\n"
-            "\nGet all transactions in blocks since block [blockhash], or all transactions if omitted\n"
+            "listsinceblock ( \"blockhash\" target_confirmations include_watchonly include_replaced )\n"
+            "\nGet all transactions in blocks since block [blockhash], or all transactions if omitted.\n"
+            "If \"blockhash\" is no longer a part of the main chain, all transactions affecting the node wallet from blockhash back to the fork point are included in the \"replaced\" array.\n"
+            "In addition, transactions from the fork point up to the tip of the active chain are included in \"transactions\".\n"
             "\nArguments:\n"
             "1. \"blockhash\"            (string, optional) The block hash to list transactions since\n"
             "2. target_confirmations:    (numeric, optional) The confirmations required, must be 1 or more\n"
             "3. include_watchonly:       (bool, optional, default=false) Include transactions to watch-only addresses (see 'importaddress')"
+            "4. include_replaced:        (bool, optional, default=true) Show transactions that were replaced due to a reorg in the \"replaced\" array"
```

Comment:
Me neither. It was mostly suggested as a way for legacy code to cope better, but only really crappy legacy code (that broke when a new key was introduced) would be affected I guess.